src/redux/Actions.ts

```diff
@@ -232,12 +192,9 @@ export const resetAll = (): RESET_ALL => ({
 });
 
 // Validates the setting object and adds missing settings if it doesn't already exist in the initialState
-export const validateSettings: ActionCreator<ThunkAction<
-  void,
-  State,
-  null,
-  ReduxAction
->> = () => (dispatch, getState) => {
+export const validateSettings: ActionCreator<
```

Comment:
I recommend next time having a PR just for prettifying the code and having a PR check for Prettier. It will also make future diffs a lot cleaner. Right now, it's hard to tell what actually changed.